CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## v5.1.0
+- Fall back to V1 when V2 authentication fails.
```

Comment:
I think it would make sense that other implementations do this also.
@prajon84  @bgong-mdsol  @ejennings-mdsol

Reply:
@jcarres-mdsol : Yes We will be making changes too. We just had discussion today.

Reply:
We fall back to V1 only when V1 signature is provided right?

Reply:
yes, that's right.